src/base/net/downloadmanager.h

```diff
@@ -67,10 +67,14 @@ namespace Net
         bool handleRedirectToMagnet() const;
         DownloadRequest &handleRedirectToMagnet(bool value);
 
+        short redirectionCounter() const;
+        DownloadRequest &incrementRedirectionCounter();
```

Comment:
It should be `DownloadRequest &redirectionCounter(short value);` instead. Please don't break used pattern.